docs/03-tutorials/00-serverless/svls-15-use-external-scalers.md

```diff
@@ -133,5 +133,76 @@ Follow these steps:
     scaled-function   serverless.kyma-project.io/v1alpha2.Function   scaled-function   5     10    cpu                         True    True     Unknown    4m15s
     ```
 
+  </details>
+  <details>
+  <summary label="keda-prometheus">
+  Keda Prometheus
+  </summary>
+
+1. Install [Keda](https://keda.sh/docs/2.8/deploy/) if it is not present on your cluster.
+
+2. Create your Function with the `replicas` value set to 1, to prevent the internal Serverless HPA creation:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha2
+    kind: Function
+    metadata:
+      name: scaled-function
+    spec:
+      runtime: nodejs14
+      replicas: 1
+      source:
+        inline:
+          dependencies: ""
+          source: |
+            module.exports = {
+              main: function(event, context) {
+                return 'Hello World!'
+              }
+            }
+    EOF
+    ```
+
+3. Create the ScaledObject resource based on, exposed by the Istio, the `istio_requests_total` metric:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: keda.sh/v1alpha1
+    kind: ScaledObject
+    metadata:
+      name: scaled-function
+    spec:
+      scaleTargetRef:
+        apiVersion:    serverless.kyma-project.io/v1alpha2
+        kind:          Function
+        name:          scaled-function
+      minReplicaCount:  1  # You can go with 0 ( scaling to zero ) in case your function is fed from messaging queue that would buffer unhandled requests or if you are fine with function downtime at cold start periods
+      maxReplicaCount:  5
+      triggers:
+      - type: prometheus
+        metadata:
+          serverAddress: http://prometheus-operated.kyma-system.svc.cluster.local:9090
+          metricName: istio_requests_total
+          query: round(sum(irate(istio_requests_total{reporter=~"source",destination_service=~"scaled-function.default.svc.cluster.local"}[2m])), 0.001)
+          threshold: '6.5'
+    EOF
+    ```
+
+    >**NOTE:** in this tutorial we use the `prometheus` trigger because of its simple configuration. If you want to use another trigger check the official [list of supported triggers](https://keda.sh/docs/2.8/scalers/).
```

Comment:
```suggestion
    >**NOTE:** This tutorial uses the `prometheus` trigger because of its simple configuration. If you want to use another trigger check the official [list of supported triggers](https://keda.sh/docs/2.8/scalers/).
```
Can you change that also in the KEDA CPU note?